src/constants/memory/wram.asm

```diff
@@ -2940,7 +2940,7 @@ wSwitchableObjectAnimationStage::
   ds 1 ; D6F8
 
 ; Unlabeled
-wD6F9::
```

Comment:
Maybe this is to indicate if you are walking on a raised switchblock?

Reply:
Yes, that seems likely. It's also used one place to play `NOISE_SFX_FOOTSTEP` when Link transitions to standing on the ground, ie. when falling down from a raised switchblock.